src/pages/search/filters/BaseFilterItem.tsx

```diff
@@ -15,7 +15,14 @@ export const BaseFilterItem = ({ title, fontWeight = 600, children }: BaseFilter
   const toggleOpen = () => setIsOpen(!isOpen);
 
   return (
-    <>
+    <Box
+      sx={{
+        m: '1rem',
+        bgcolor: 'background.default',
+        borderRadius: '10px',
+        border: '2px solid',
+        borderColor: 'primary.main',
```

Comment:
kan dette gjøres på 1 linje? `border: '2px solid primary.main'`

Reply:
dessverre ikke :( samme som at man må bruke `bgcolor` i stedet for `backGroundColor` for å kunne bruke theme direkte

Reply:
bummer :/